src/components/MyAccount/Settings/AccountSettingsTab.test.tsx

```diff
@@ -2,21 +2,35 @@ import { patron } from "../../../../__test__/fixtures/rawSierraAccountData"
 import AccountSettingsTab from "./AccountSettingsTab"
 import MyAccount from "../../../models/MyAccount"
 import { render, screen } from "../../../utils/testUtils"
+import * as helpers from "../../../../pages/api/account/helpers"
+import userEvent from "@testing-library/user-event"
+
+jest.spyOn(helpers, "updatePatronSettings")
 
 describe("AccountSettingsTab", () => {
+  global.fetch = jest
+    .fn()
+    .mockResolvedValueOnce({
+      json: async () => console.log("updated"),
+      status: 200,
+    } as Response)
+    .mockResolvedValueOnce({
+      json: async () => console.log("not updated"),
+      status: 500,
+    } as Response)
+
   it("can render a complete patron", () => {
     const myAccountPatron = MyAccount.prototype.buildPatron(patron)
     render(<AccountSettingsTab settingsData={myAccountPatron} />)
-    ;[
-      "streganonna@gmail.com",
-      "Phone",
-      "Notification preference",
-      "Home library",
-      "Pin/Password",
-    ].forEach((patronInfo) => {
-      const element = screen.getByText(patronInfo)
-      expect(element).toBeInTheDocument()
-    })
+
+    const email = screen.getByText("streganonna@gmail.com")
```

Comment:
The test has a mix of patron values and labels. So it should test the patron's phone number just as it tests for the patron's email value. Same for home library but for pin/password, `****` can be used.

Reply:
updated to check for values.